src/radicalpy/classical.py

```diff
@@ -19,6 +19,38 @@ def get_delta_r(mutual_diffusion: float, delta_T: float) -> float:
     return np.sqrt(6 * mutual_diffusion * delta_T)
 
 
+def kinetics(
```

Comment:
@lmantill We have too many things called `kinetics`. I would call this `kinetics_solver`, would that be ok?

Reply:
`kinetics_solver` is good